.gitignore

```diff
@@ -1 +1,4 @@
 *.pyc
+config.py
+.vscode
```

Comment:
Looks good, although could you move `.vscode` to your user git ignore file (`~/.config/git/ignore`)? As it isn't specific to the project. Ta muchly